lib/plausible/plugins/api/tokens.ex

```diff
@@ -39,17 +39,31 @@ defmodule Plausible.Plugins.API.Tokens do
 
   @spec delete(Site.t(), String.t()) :: :ok
   def delete(site, token_id) do
-    Repo.delete_all(from t in Token, where: t.site_id == ^site.id and t.id == ^token_id)
+    Repo.delete_all(from(t in Token, where: t.site_id == ^site.id and t.id == ^token_id))
     :ok
   end
 
   @spec list(Site.t()) :: {:ok, [Token.t()]}
   def list(site) do
-    Repo.all(from t in Token, where: t.site_id == ^site.id, order_by: [desc: t.id])
+    Repo.all(from(t in Token, where: t.site_id == ^site.id, order_by: [desc: t.inserted_at]))
```

Comment:
```suggestion
    Repo.all(from(t in Token, where: t.site_id == ^site.id, order_by: [desc: t.inserted_at, desc: t.id]))
```

just to avoid ambiguity, perhaps (timestamps still have default resolution of seconds AFAIK)?